src/oc/web/components/rich_body_editor.cljs

```diff
@@ -86,6 +86,7 @@
   (let [editable (or editable (get-media-picker-extension s))]
     (.saveSelection editable))
   (dis/dispatch! [:input [:media-input :media-video] true])
+  (dis/dispatch! [:input [:cmail-data :has-changes] true])
```

Comment:
Why you set cmail has-changes here? This lines is showing the overlay to collect the video url (`media-video-modal` component).

The add video and add images inside the ME actions always end up changing the body of the entry, that HTML node has a listener that set the `has-change` when something changes inside it.

I don't think we need to make sure we autosave here, maybe when `media-video-add` and `media-photo-add-if-finished` are called but as i said the body automatically set has-changes to true when something changes inside.

Why did you add this here?